examples/traefik/librenms.env

```diff
@@ -14,3 +14,5 @@ LIBRENMS_SNMP_COMMUNITY=librenmsdocker
 
 LIBRENMS_WEATHERMAP=false
 LIBRENMS_WEATHERMAP_SCHEDULE=*/5 * * * *
+
+SNMP_TRAP_MIB_DIRS=/opt/librenms/mibs:/opt/librenms/mibs/cisco
```

Comment:
Please remove changes from `librenms.env`, this is just an example and majority of users will prefer the default value.

Reply:
Changes should reflect above. All example .env files removed reference. Readme updated. Env Var name updated to be more specific.